### DISCUSSION TITLE
Fllater

### DISCUSSION THREAD

**Discussion by ahmadgharbia** [community/community#144872] [Programming Help]
### Body

في حد بقد يساعدني بهاي المشكله المتعلقه بالكود
![Screenshot_٢٠٢٤-١١-١٧-٢٢-٢٣-٠٥-٥٦٢-edit_com miui gallery](https://github.com/user-attachments/assets/39d54800-4ef0-4e16-898b-1648117185a9)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Kazakio** [comment]
مرحبًا أحمد،

المشكلة تتعلق بالحزمة com.arthenica.ffmpegkit.flutter في ملف AndroidManifest.xml. جرب هذه الخطوات:
	1.	افتح الملف واحذف أو عدّل الحزمة المذكورة.
	2.	شغل الأوامر:

```
flutter clean  
flutter pub get  
```


3.	تأكد من أن المكتبة محدثة بإصدار متوافق، أو أزلها وأعد تثبيتها:

```
flutter pub remove ffmpegkit_flutter_min_gpl  
flutter pub add ffmpegkit_flutter_min_gpl  

```


إذا استمرت المشكلة، شارك التفاصيل! بالتوفيق.

**Comment by mecodeatlas** [comment]
Hi @ahmadgharbia, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
